### ISSUE TITLE
For select lists, why is the "multiple" option not announced?

### ISSUE THREAD

**Issue by jmuheim** [nvaccess/nvda#7579]
When focusing a `<select>` element, NVDA doesn't announce whether only one or multiple options can be selected:

https://codepen.io/accessibility-developer-guide/full/PKoRLG/

Is this a bug?

**Comment by jmuheim** [comment]
Any news on this?

**Comment by Adriani90** [comment]
cc: @jcsteh

**Comment by jcsteh** [comment]
Firefox (and I presume other browsers) do expose this information, but NVDA doesn't report it at this point. I've never been convinced this is necessary - I'm not even sure how a sighted user immediately knows that a list is multi-selectable without selecting things first - but I'll leave that decision to others.

**Comment by mdlawler** [comment]
I believe that 18409 fixes this.

**Comment by LeonarddeR** [comment]
Good catch @mdlawler, this is indeed fixed by #18409
